recipes/methyldackel/meta.yaml

```diff
@@ -1,20 +1,22 @@
 package:
   name: methyldackel
-  version: "0.3.0"
+  version: "0.4.0"
 
 build:
-  number: 3
+  number: 0
 
 source:
-  sha256: b5a49f213bba9113d2c9951de9daf676efd9b9c94327097e960b8623765f1b91
-  url: https://github.com/dpryan79/MethylDackel/archive/0.3.0.tar.gz
+  sha256: eea3fa5167609ca5a293a2be7c4ad29566aad84a99e7d14d2991685071cfed2e
+  url: https://github.com/dpryan79/MethylDackel/archive/0.4.0.tar.gz
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
+    - htslib
     - zlib
   run:
+    - htslib
```

Comment:
@dpryan79 try to remove htslib and zlib from run dependencies. It should work with run_dependencies.

Reply:
What would the benefit of that be and is that actually documented somewhere?